docs/requirements.txt

```diff
@@ -2,3 +2,4 @@ sphinx-tabs
 cmake
 sphinx_rtd_theme
 sphinx~=5.3.0
+sphinx_exec_code~=0.8
```

Comment:
SimpleITK is also going to be needed here.

Reply:
Right. Should I specify a version? Maybe >=2.2 ?

Reply:
How about "~=2.2.0"

Reply:
Okay, but that wouldn't match 2.3 and above, right? So would that require an update this requirements.txt each time we push a minor upgrade?

Reply:
Yes. But it would also ensure if an incompatible change was made the older versions could still be built.